Upstream: Not applicable
Reason: Make it not depend on certifi, we patch system certificates in

diff --git a/setup.py b/setup.py
index 2da9ba0..5147636 100755
--- a/setup.py
+++ b/setup.py
@@ -44,8 +44,7 @@ packages = ['requests']
 requires = [
     'chardet>=3.0.2,<4',
     'idna>=2.5,<3',
-    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1',
-    'certifi>=2017.4.17'
+    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1'
 
 ]
 test_requirements = [
